src/sentry/api/bases/team.py

```diff
@@ -17,19 +17,16 @@ class TeamPermission(OrganizationPermission):
     }
 
     def has_object_permission(self, request, view, team):
-        result = super(TeamPermission, self).has_object_permission(
-            request, view, team.organization)
+        result = super(TeamPermission,
```

Comment:
might be configuration to make this slightly nicer, but dont care as long as i dont have to think about choosing one way or the other